source/assets/sass/components/_docsearch.scss

```diff
@@ -4,3 +4,7 @@
   justify-content: center;
   height: 100%;
 }
+
+.DocSearch-Button-Placeholder {
+  color: #4A4F4F;
```

Comment:
Is there a named brand color we can use for this?

Reply:
Updated!

Reply:
I meant one of the brand colors from [the style guide](https://sass-lang.com/styleguide/color/).

Reply:
Ah got it thank you-- updated!